pkg/k8shandler/defaults.go

```diff
@@ -0,0 +1,29 @@
+package k8shandler
+
+import (
+	"fmt"
+)
+
+const (
+	modeUnique    = "unique"
+	modeOpsShared = "ops_shared"
+	defaultMode   = modeUnique
+)
+
+func KibanaIndexMode(mode string) (string, error) {
+	if mode == "" {
+		return defaultMode, nil
+	}
+	if mode == modeUnique || mode == modeOpsShared {
+		return mode, nil
+	}
+	return "", fmt.Errorf("invalid kibana index mode provided [%s]", mode)
+}
+
+func EsUnicastHost(clusterName string) string {
+	return clusterName + "-cluster"
+}
```

Comment:
@ewolinetz is this how it works?

I'm currently working with https://github.com/openshift/cluster-logging-operator/blob/master/pkg/k8shandler/logstore.go#L103-L117

Reply:
This operator creates that service btw

Reply:
@ewolinetz  we should be safe then, aye? https://github.com/openshift/elasticsearch-operator/blob/master/pkg/k8shandler/services.go#L18